WordPress/src/main/java/org/wordpress/android/ui/activitylog/list/filter/ActivityLogTypeFilterFragment.kt

```diff
@@ -31,6 +39,35 @@ class ActivityLogTypeFilterFragment : DialogFragment() {
     }
 
     override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
-        return inflater.inflate(layout.activity_log_type_filter_fragment, container, false)
+        return inflater.inflate(R.layout.activity_log_type_filter_fragment, container, false)
+    }
+
+    override fun getTheme(): Int {
```

Comment:
1) Minor: This function can be converted to an expression body, thus winning us a couple of lines.
1) Minor: I also suggest moving the `getTheme` above the `onCreateView(...)` function for the same reasons I suggested the same for the adapter in my previous comment.

Reply:
Fixed in 559082bd9999f98a991a84c76fbd8be6ac542b8c